core/src/main/java/com/sequenceiq/cloudbreak/service/ServiceEndpointCollector.java

```diff
@@ -193,7 +204,7 @@ public String getManagerServerUrl(Cluster cluster, String managerIp) {
             if (ExposedService.HIVE_SERVER.equals(exposedService) || ExposedService.HIVE_SERVER_INTERACTIVE.equals(exposedService)) {
                 return getHiveJdbcUrl(gateway, managerIp);
             } else if (ExposedService.NAMENODE.equals(exposedService) && versionComparator.compare(() -> stackVersion, () -> "2.6") > 0) {
```

Comment:
I guess version `"2.6"` refers to the legacy HDP stack. 😃 So this just happens to work because CDH / CDP will have a higher version number.

Reply:
@pdarvasi can you please fix this one also?

Reply:
I don't think this causes any problem except for the original code being ugly, but fixed anyways.